e2e/portalicious/pages/BasePage.ts

```diff
@@ -93,6 +97,13 @@ class BasePage {
   async openCreateNewProject() {
     await this.page.getByRole('button', { name: 'Add project' }).click();
   }
+
+  async FileChooserComponent(filePath: string) {
```

Comment:
I meant this more like a separate file, similar to `TableComponent`. And even then, it shouldn't be on all pages, but only on the pages with a File chooser component.

That said, maybe it doesn't warrant a separate component (up to you), but if not, then I would at a minimum rename this function

```suggestion
  async chooseAndUploadFile(filePath: string) {
```

Reply:
I just changed the function name for now. I do not know if indeed we need to create the separate component for it. Sorry my brain was somewhere else and I did not understand what you meant.